configs/common/packages.yaml

```diff
@@ -152,6 +156,8 @@
       version: ['5.1.0']
     mpich:
       variants: ~hwloc +two_level_namespace
+    mysql:
+      variants: +download_boost
```

Comment:
I am not sure how relevant this is at the moment, given that we install mysql as external dependency on all systems, but it won't hurt either.